lib/iris/analysis/calculus.py

```diff
@@ -471,6 +471,9 @@ def curl(i_cube, j_cube, k_cube=None):
     Calculate the 2-dimensional or 3-dimensional spherical or cartesian
     curl of the given vector of cubes.
 
+    The cube standard names must match one the combinations in
```

Comment:
one of the

Reply:
159e53c6b59c1eb8e7506dbe6bd01764665baa8f